zk_coordinator.go

```diff
@@ -1152,7 +1152,7 @@ func ZookeeperConfigFromFile(filename string) (*ZookeeperConfig, error) {
 	setStringSliceConfig(&config.ZookeeperConnect, z["zookeeper.connect"], ",")
 	setStringConfig(&config.Root, z["zookeeper.kafka.root"])
 
-	if err := setDurationConfig(&config.ZookeeperTimeout, z["zookeeper.connection.timeout"]); err != nil {
+	if err := setDurationConfig(&config.ZookeeperSessionTimeout, z["zookeeper.connection.session_timeout"]); err != nil {
```

Comment:
can we please make this `zookeeper.connection.session.timeout` to be consistent?